data/mods/TropiCataclysm/monsters/tropical_fish.json

```diff
@@ -153,7 +153,7 @@
     "id": "mon_fish_tucunare",
     "type": "MONSTER",
     "copy-from": "mon_fish_large",
-    "name": { "str": "tucunare" },
+    "name": { "str": "tucunaré" },
```

Comment:
You need to manually specify plural form for this one.

Reply:
Hm, so it doesn't add plurals if it doesn't end with the regular character then?

Reply:
The failed test says "Cannot autogenerate plural form.  Please specify the plural form explicitly."